test/functions_test.rb

```diff
@@ -151,10 +162,14 @@ def nice_color_argument(color)
         return Script::String.new(color.to_s, :string)
       end
 
-      def returns_a_color()
+      def returns_a_color
         return Script::Color.new(red: 0, green: 0, blue: 0)
```

Comment:
This was just a mistake i made early on.  I have started down the path of rectifying it by aliasing to both the incorrect module, as well as the correct one.  (see above where i alias the SassC types)